lib/thrift/generator.ex

```diff
@@ -94,6 +100,20 @@ defmodule Thrift.Generator do
     end
   end
 
+  defp generate_const_modules(%Schema{constants: constants})
+  when constants == %{} do
```

Comment:
Nope :/  `%{}` matches any map, this is the only way I know of to get an empty map through pattern matching.